src/test/java/calculator/InputVoTest.java

```diff
@@ -0,0 +1,37 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class InputVoTest {
+
+    @DisplayName("null, 공백 체크")
+    @Test
+    void validateNull() {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    @NullAndEmptySource
    @ValueSource(strings = {" "})
    void validateNull(String input) {
```